mycroft/res/ui/SeekControl.qml

```diff
@@ -0,0 +1,272 @@
+                            Rectangle {
+                                anchors {
+                                    left: parent.left
+                                    top: parent.top
+                                    bottom: parent.bottom
+                                }
+                                radius: height
+                                color: Kirigami.Theme.highlightColor
+                                width: slider.position * (parent.width - slider.handle.width/2) + slider.handle.width/2
+                            }
+                        }
+
+                        Controls.Label {
+                            anchors {
+                                left: parent.left
+                                top: groove.bottom
+                                topMargin: Kirigami.Units.smallSpacing
+                            }
+                            horizontalAlignment: Text.AlignLeft
+                            verticalAlignment: Text.AlignVCenter
+                            text: formatTime(playPosition)
+                            color: "white"
+                        }
+
+                        Controls.Label {
+                            anchors {
+                                right: parent.right
+                                top: groove.bottom
+                                topMargin: Kirigami.Units.smallSpacing
+                            }
+                            horizontalAlignment: Text.AlignRight
+                            verticalAlignment: Text.AlignVCenter
+                            text: formatTime(duration)
+                        }
+                    }
+                KeyNavigation.up: video
+                KeyNavigation.left: button
+                Keys.onReturnPressed: {
+                    hideTimer.restart();
+                    if(!navSliderItem){
+                            navSliderItem = true   
+                        } else {
+                            navSliderItem = false
+                        }
+                    }
+                
+                Keys.onLeftPressed: {
+                        console.log("leftPressedonSlider")
+                        hideTimer.restart();
+                        if(navSliderItem) {
+                            video.seek(video.position - 5000)
+                        } else {
+                            button.forceActiveFocus()
+                        }
+                }
+                
+                Keys.onRightPressed: {
+                        hideTimer.restart();
+                        if(navSliderItem) {
+                            video.seek(video.position + 5000)
+                        }
+                    }
+                }
+
+            }
+        }
+    }
+    
+
+    function formatTime(timeInMs) {
+        if (!timeInMs || timeInMs <= 0) return "0:00"
+        var seconds = timeInMs / 1000;
+        var minutes = Math.floor(seconds / 60)
```

Comment:
Should we add in an `hours` that is only shown if time > 1hr. 

Would also need to pad the minutes `if (hours > 0 && minutes < 10)`

Reply:
still need to implement this, If i understand correctly an hour would be represented like "1:05:04" "H:M:S"  format, and on duration change once it changes below an hour on displayed video would still be represented like "0:50:04" "H:M:S" format or "50.04" "M:S" format ?

Reply:
Yeah, good question, I hadn't thought of that but I like the idea of keeping the "H:M:S" format if the total duration is > 1hr but the time remaining drops below 1:00:00.
Where as if the total duration was < 1hr it would only ever show the "M:S" format. 

I think it's less cognitive load for the user, it indicates that we're under an hour to go, and less chance of strange things happening when rendering the time remaining to screen. 

@dschweppe do you have an opinion?